application/config/routes.php

```diff
@@ -58,8 +58,8 @@
 $route['datagov/(:any)/(:any)'] = "campaign/$1/$2";
 $route['datagov/(:any)/(:any)/(:any)'] = "campaign/$1/$2/$3";
 
-$route['offices/(:any)'] = "offices/routes/$1";
-$route['docs/(:any)'] = "docs/routes/$1";
+$route['offices/(.+)'] = "offices/routes/$1";
+$route['docs/(.+)'] = "docs/routes/$1";
```

Comment:
Can we rewrite these with `:any` as they are above?

This is a code smell to me, one controlling handling several routes. Even though we're not actually solving anything by using `:any`, we're calling out that the Offices controller is handling 5 different routes (or maybe 16 depending on how you count :grimacing: )

Reply:
I was just trying to avoid detangling the controllers as part of achieving the upgrade; your suggestion is a good half-step, so sure, I'll do that.